jena-arq/src/test/java/org/apache/jena/sparql/expr/TestFunctions2.java

```diff
@@ -165,6 +167,23 @@
     public void substr_21()         { test("substr('', 'one', 3)",            "''") ; }
     @Test(expected=ExprEvalException.class)
     public void substr_22()         { test("substr('', 1, 'three')",            "''") ; }
+    
+    // Codepoint outside UTF-16. 
+    // These are  U+0001F46A 👪 - FAMILY
```

Comment:
Emoticons being useful for testing :tada:

Reply:
:-)

I didn't put them raw in the tests themselves because of portability concerns over binary/text.
I did test with 👪.
(ARQ is not handling \U in SPARQL properly yet.)